onnx/defs/nn/defs.cc

```diff
@@ -314,13 +314,20 @@ OPERATOR_SCHEMA(Flatten)
     .NumInputs(1)
     .NumOutputs(1)
     .SetDoc(R"DOC(
-Flattens the input tensor into a 2D matrix, keeping the first dimension
-unchanged.
+Flattens the input tensor into a 2D matrix. If input tensor has shape
+(d_1, d_2, ... d_n) then the output will have shape
+(d_1 X d_2 ... d_(axis-1), d_axis X d_(axis+1) ... X dn).
 )DOC")
-    .Input(0, "input", "A tensor of rank >= 2.")
+    .Input(0, "input", "A tensor of rank > axis.")
     .Output(
         0,
         "output",
-        "A tensor of rank 2 with the contents of the input tensor, "
-        "with first dimension equal first dimension of input, and remaining "
-        "input dimensions flattened into the inner dimension of the output.");
+        "A 2D tensor with the contents of the input tensor, "
+        "with input dimensions up to axis flattened to the outer dimension "
+        "of the output and remaining input dimensions flattened into the inner "
+        "dimension of the output.")
+    .Attr(
+        "axis",
+        "(Default to 1) Indicate up to which input dimensions "
```

Comment:
A naïve question: is axis numbering zero-based or one-based? The preceding lines (318,319) start with d_1, but this line implies zero-based. Doesn't one of them have to be changed?

Reply:
Oh it should be zero based, will update the doc.